### ISSUE TITLE
Error message format is not correct at timeout

### ISSUE THREAD

**Issue by dogfootman** [cloud-barista/cb-tumblebug#1300]
**What happened**
: I looked up the MCIS list, but there was a Timeout error.

/ns/ns01/mcis
```
{
    "a": "<html><head><title>Timeout</title></head><body><h1>Timeout</h1></body></html>"
}
```
**What you expected to happen**
: 
It should be shown as below
{
  "message": "Any message"
}

**Comment by seokho-son** [comment]
Hi @dogfootman 
Thank you for this bug report. I guess the message is the default message for [timeout  middle-ware](https://github.com/cloud-barista/cb-tumblebug/blob/e36988a30ce7a5d1e61f2e3fe0db5430cb822b57/src/api/rest/server/server.go#L179) from Echo framework.
Let me check the default message, and update if necessary.

**Comment by seokho-son** [comment]
Hi @dogfootman 

This issue will be resolved by #1314 
Thank for the report.

> ErrorMessage: "Error: request time out (60s)"
